android/gradle.properties

```diff
@@ -1,15 +1,18 @@
-## Project-wide Gradle settings.
-#
-# For more details on how to configure your build environment visit
-# http://www.gradle.org/docs/current/userguide/build_environment.html
-#
-# Specifies the JVM arguments used for the daemon process.
-# The setting is particularly useful for tweaking memory settings.
-# Default value: -Xmx10248m -XX:MaxPermSize=256m
-# org.gradle.jvmargs=-Xmx2048m -XX:MaxPermSize=512m -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF-8
-#
-# When configured, Gradle will run in incubating parallel mode.
-# This option should only be used with decoupled projects. More details, visit
-# http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
-# org.gradle.parallel=true
-#Thu May 12 17:26:12 PDT 2016
+VERSION_CODE=2
+VERSION_NAME=0.7.1
+GROUP=com.airbnb.android
+
+POM_DESCRIPTION=React Native Map view component for Android
+POM_URL=https://github.com/lelandrichardson/react-native-maps/tree/new-scv
```

Comment:
I believe this doesnt matter much, it's only used by maven central, but since the lib will be pulled from node_modules, it's never used